tests/acceptance/preprocessor-smoke-test-slow.js

```diff
@@ -67,7 +67,7 @@ describe('Acceptance: preprocessor-smoke-test', function() {
       });
   });
 
-  it.only('addon registry entries are added in the proper order', function() {
+  it('addon registry entries are added in the proper order', function() {
```

Comment:
I think line 83 needs to be:

```
return runCommand(path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember'), 'build', '--silent');
```

But this test still keeps exiting with 1 when ran end to end.